### ISSUE TITLE
[Feature?]: players should be able to send private notifications to DM

### ISSUE THREAD

**Issue by caseybaggz** [virtual-dnd/virtual-dnd#15]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

In order to support all games being a11y to all users and prevent the DM from telling stories that might be triggering. Each player should have a widget in the dashboard that:

- [ ] Allows the to choose from a visual smile-face list if there is a story issue that could be triggering in some way
- [ ] Notify the DM when that is the case
- [ ] The DM should privately receive a toast notification of trigger with a message recommending to pivot the story
